editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -25,19 +25,32 @@ interface MapContainerProps {
   environment: PreviewEnvironment;
 }
 
+/**
+ * Generate a style which increases the map size as the window grows
+ * and maintains a consistent right margin
+ */
+const dynamicMapSizeStyle = (theme: Theme): Record<string, any> => {
+  const mainContainerWidth = `${theme.breakpoints.values.md}px`;
+  const mainContainerMargin = `((100vw - ${mainContainerWidth}) / 2)`;
+  const mapMarginRight = "100px";
+
+  const style = {
+    [theme.breakpoints.up("md")]: {
+      height: "70vh",
+      width: `calc(${mainContainerMargin} + ${mainContainerWidth} - ${mapMarginRight})`,
+    },
+  };
+
+  return style;
+};
```

Comment:
This style could also be more terse, but I personally think the more explicit is more understandable.

```js
width: `calc(${theme.breakpoints.values.md}px + ((100vw - ${theme.breakpoints.values.md}px) / 2) - 100px`,
```